ovs/dal/tests/test_alba.py

```diff
@@ -112,87 +112,124 @@ def test_localstack(self):
+                                                   'partition_amount': 0,
+                                                   'size': 1024 ** 3,
+                                                   'state': 'empty',
+                                                   'state_detail': '',
+                                                   'usage': {}}},
+                     'get_metadata': {'_version': 3}}
+        stack = copy.deepcopy(get_stack)['get_stack']['alba_slot_1']
+        _move(stack)  # Do mapping
+        ASDManagerClient.test_results[node] = get_stack
+        expected = {'alba_slot_1': stack}
         node.invalidate_dynamics()
-        self.assertDictEqual(alba_backend._local_stack(), expected)
+        self.assertDictEqual(node._stack(), expected)
 
         # Validate initialized disks
-        ASDManagerClient.test_results[node]['get_stack']['alba_disk_1'].update({'available': False,
-                                                                                'mountpoint': '/mnt/alba-asd/disk-1'})
-        expected['node_1']['alba_disk_1'].update({'mountpoint': '/mnt/alba-asd/disk-1',
-                                                  'available': False})
+        updated_osd_values = {'asd_id': 'alba_osd_1',
+                              'osd_id': 'alba_osd_1',
+                              'metadata': None,
+                              'capacity': 1024 ** 3,
+                              'folder': 'alba_osd_1',
+                              'home': '/mnt/alba-asd/disk-1/alba_osd_1',
+                              'ips': ['127.0.0.1'],
+                              'log_level': 'info',
+                              'multicast': None,
+                              'node_id': 'node_1',
+                              'port': 35001,
+                              'rocksdb_block_cache_size': 8 * 1024 ** 2,
+                              'state': 'ok',
+                              'transport': 'tcp',
+                              'type': 'ASD'}
+        updated_stack_values = {'available': False,
+                                'mountpoint': '/mnt/alba-asd/asd_1',
+                                'state': 'ok',
+                                'partition_amount': 1,
+                                'partition_aliases': ['/dev/disk/by-id/alba_disk_1_partition_1'],
+                                'osds': {'alba_osd_1': updated_osd_values},
+                                'usage': {'available': 1024 ** 3 - 1024,
+                                          'size': 1024 ** 3,
+                                          'used': 1024}}
+
+        expected_updated_osds = copy.deepcopy(updated_osd_values)
+        _move(expected_updated_osds)  # Map state <-> status
+        expected_values_stack = copy.deepcopy(updated_stack_values)
+        _move(expected_values_stack)  # Map state <-> status
+        expected_updated_osds.update({'claimed_by': None})  # Osd not in model and is not claimed no None
```

Comment:
The dict that is presented is the data the mocked asd-manager will return.
Afterwards I copy this data to avoid retyping it, change a few keys and add my expected values to it.
I will document it to make it more clear.